pandas/tseries/frequencies.py

```diff
@@ -38,18 +38,20 @@ class FreqGroup(object):
     FR_NS = 12000
 
 
-US_RESO = 0
-MS_RESO = 1
-S_RESO = 2
-T_RESO = 3
-H_RESO = 4
-D_RESO = 5
+NS_RESO = 0
```

Comment:
so would be nice to clean this up. This is essentially an Enum, but that only exists in 3.5 (maybe we could add the backport of https://pypi.python.org/pypi/enum34 as an in-inline package), not sure this is worth it though.

In any event just put these in a list and ``enumerate`` if needed.